deepchem/models/torch_models/run_bert_model.py

```diff
@@ -0,0 +1,47 @@
+from deepchem.feat.bert_tokenizer import BertFeaturizer
```

Comment:
This looks like a script file which we can't merge into deepchem :). Only library models should go in the main trunk of the repo and not script fiels

Reply:
This code should probably be migrated to unit tests